appveyor-linux-binary.yml

```diff
@@ -0,0 +1,296 @@
+version: 1.0.{build}
```

Comment:
There are too many `appveyor*` files at the root level, we might want to move them under `.appveyor` folder.

And there is some level of duplication between these files, I wonder if we can use some sort of "import" and move common configuration into separate files.